### ISSUE TITLE
Support prefers-reduced-motion for skeleton states

### ISSUE THREAD

**Issue by carmacleod** [carbon-design-system/carbon#2440]
Some people feel sick when they experience unexpected motion on the web.
The skeleton states animations may be a problem for these people.
There's a relatively new [media query called prefers-reduced-motion](https://caniuse.com/#search=prefers-reduced-motion) that was introduced by Apple in Safari, and is supported in Firefox, and soon to be supported in Chrome.
Please consider supporting this so that if a user chooses the "Reduce Motion" setting in their browser/OS, the skeleton states remain static and they do not animate.

- [CSS Tricks Article: _"An Introduction to the Reduced Motion Media Query"_](https://css-tricks.com/introduction-reduced-motion-media-query/)
- [MDN Documentation](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-reduced-motion)
  - including an example that is supported in Safari, Firefox, and the [upcoming Chrome 74](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-reduced-motion)

**Comment by dakahn** [comment]
Not stale. I'd like to get this in soon. Upping the severity as well.

@carbon-design-system/design

**Comment by aagonzales** [comment]
Seems like a good idea. @dakahn @joshblack what specifically do you need from design to move forward on this?

**Comment by shixiedesign** [comment]
Yes but enhancement. Will backlog for July project cycle.

**Comment by joshblack** [comment]
@aagonzales I believe we just want agreement that this is something we'd want to support on the design side, meaning that this is a valid way to represent the Carbon Design System for this user preference (reduced motion).

On the dev side, this would mean that any motion token or curve defaults to instant when this preference is enabled.

**Comment by carmacleod** [comment]
In case it's helpful: a recent post about prefers-reduced-motion:
https://www.scottohara.me/note/2019/07/12/reduced-motion-video.html

**Comment by dakahn** [comment]
This is an important enhancement. Thanks for sharing that link @carmacleod
